docs/content/docs/wip/ingress.md

```diff
@@ -0,0 +1,447 @@
+    kind: Ingress
+    metadata:
+      name: httpbin-ingress
+      namespace: httpbin
+      annotations:
+        kubernetes.io/ingress.class: nginx
+    spec:
+      rules:
+      - http:
+          paths:
+          - path: "/status/200"
+            backend:
+              serviceName: httpbin
+              servicePort: 14001
+    ```
+
+    Confirm that the httpbin-ingress has been successfully deployed. 
+    
+    ```console
+    $ kubectl get ingress -n httpbin
+    NAME              CLASS    HOSTS   ADDRESS         PORTS   AGE
+    httpbin-ingress   <none>   *       20.72.132.186   80      11h
+    ```
+
+1. Confirm that a request to the httpbin service from the external IP address of the Ingress resource succeeds (in this case, the external address would be `20.72.132.186`)
+
+    ```bash
+    curl http://<external-ip>/status/200
+    ```
+
+1. Update the existing ingress with a host specified by applying the following yaml, and confirm that the request succeeds:  
+
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+      name: httpbin-ingress
+      namespace: httpbin
+      annotations:
+        kubernetes.io/ingress.class: nginx
+    spec:
+      rules:
+      - host: httpbin.com 
+        http:
+          paths:
+          - path: /status/200
+            backend:
+              serviceName: httpbin
+              servicePort: 14001
+    ```
+
+    ```bash
+    curl http://<external-ip>/status/200 -H "Host: httpbin.com"
+    ```
+
+
+## Sample Ingress Configurations
+
+The following section describes sample ingress configurations used to expose services managed by OSM outside the cluster. The configuration might differ based on the ingress controller being used.
+
+The example configurations describe how to expose HTTP and HTTPS routes for the `httpbin` service running on a pod with the service account `httpbin` on port `14001` in the `httpbin` namespace, outside the cluster. The ingress configuration will expose the HTTP path `/status/200` on the `httpbin` service.
+
+Since OSM uses its own root certificate, the ingress controller must be provisioned with OSM's root certificate to be able to authenticate the certificate presented by backend servers when using HTTPS ingress. With `Tresor` as the certificate provider, OSM stores the CA root certificate in a Kubernetes secret named `osm-ca-bundle` with the key `ca.crt` in the namespace OSM is deployed (`osm-system` by default). When using other certificate providers such as `cert-manager.io` or `Hashicorp Vault`, the `osm-ca-bundle` secret must be created by the user with the base64 encoded root certificate stored as the value to the `ca.crt` attribute in the secret's data.
+
+### Prerequisites
+- Install [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/deploy/#installation-guide)
+
+An ingress configuration yaml with [Nginx Ingress Controller][2] for the `httpbin` service described above would look as follows.
```

Comment:
indentation levels of these bullets are a lil confusing to me

Reply:
I don't think the line on 199 is necessary. I can remove that.